src/components/cell-control/select.jsx

```diff
@@ -16,6 +17,11 @@ const Select = forwardRef(function Select(props, ref) {
       ref={refs.container}
     >
       <SelectControl
+        classNames={{
```

Comment:
Same question as on the `AutoCompleter` control.

Reply:
As-needed basis for now.